xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1171 @@
+       up two &sls; 15 machines</link> one on the internal network and one on
+       the airgapped network.
+      </para>
+     </step>
+     <step>
+      <para>
+       Make sure you have
+       <link xlink:href="https://www.suse.com/documentation/sles-15/book_sles_docker/data/preparation.html">enabled
+       the <filename>Containers</filename> module</link> on each server.
+      </para>
+     </step>
+     <step>
+      <para>
+       You will need SSL/TLS certificates to secure services on each server.
+       Make sure the certificates are available system wide so they can be used
+       by the deployed components.
+      </para>
+     </step>
+    </procedure>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.requirements.network">
+    <title>Networking</title>
+    <para>
+     All members of the &productname; cluster must be able to communicate with
+     the internal mirror server(s) within the airgapped network.
+    </para>
+    <para>
+     Ports:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       80 HTTP - &rmtool; Server and Helm chart repository mirror
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       5000 HTTPS - Container image registry
+      </para>
+     </listitem>
+    </itemizedlist>
+    <para>
+     The external mirror server must be able to communicate with upstream
+     sources on the internet. The minimal required ports are <literal>80</literal>
+     and <literal>443</literal>.
+    </para>
+    <note>
+     <title>Additional Port Configuration</title>
+     <para>
+      If you choose to add more container image registries to your internal
+      network, these must run on different ports than the standard registry
+      running on <literal>5000</literal>. Configure your network to allow for
+      this communication accordingly.
+     </para>
+    </note>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.requirements.storage">
+    <title>Trusted Storage</title>
+    <para>
+     Transferring data from the external network mirror to the internal mirror
+     can be performed in many ways. The most common way is portable storage (USB
+     keys or external hard drives).
+    </para>
+    <para>
+     Sizing of the storage is dependent on the number of data sources that need
+     to be stored. Container images can easily measure several Gigabytes per
+     item; although they are generally smaller for &kube; related
+     applications. The overall size of any given RPM repository is at least tens
+     of Gigabytes. For example: At the time of writing, the package repository
+     for &sls; contains approximately <literal>36 GB</literal> of data.
+    </para>
+    <para>
+     The storage must be formatted to a filesystem type supporting files larger
+     than <literal>4 GB</literal>.
+    </para>
+    <para>
+     The type of filesystem used for the storage depends on your requirements.
+     We recommend <literal>btrfs</literal>.
```

Comment:
honestly the `btrfs` suggestion still bugs me especially when talking about external/usb storage.

Reply:
Then what is the recommended filesystem? As said before I don't want to have to explain how to install ntfs drivers or some other stuff. We need >4GB binary support. We can just recommend ext4? @flavio ?

Reply:
I would say that you don't really have to recommend any fs. If you recommend one, then people might start asking on how to configure it properly. I would leave this as an exercise to the deployer.

Reply:
I agree with @hwoarang

Reply:
I removed the suggestion for FS type.